app/common/state/bookmarkFoldersState.js

```diff
@@ -74,11 +74,10 @@ const bookmarkFoldersState = {
     return state
   },
 
-  editFolder: (state, editKey, folderDetails) => {
+  editFolder: (state, editKey, oldFolder, folderDetails) => {
     state = validateState(state)
-    const oldFolder = bookmarkFoldersState.getFolder(state, editKey)
 
-    if (oldFolder.isEmpty()) {
+    if (oldFolder == null) {
```

Comment:
Will this condition ever get hit (null/undefined being passed in)? The caller isn't checking for null/undefined before calling `.isEmpty()` on the param